.github/workflows/test.yml

```diff
@@ -84,7 +84,7 @@ jobs:
         run: |
           pip install --disable-pip-version-check --upgrade pip setuptools==63.* wheel scikit-build 
           pip install ${{ matrix.numpy }}
-          pip install -e .${{ matrix.extras }} --no-build-isolation ${{ matrix.pins }} pygdal==$(gdal-config --version).*
+          pip install -e .${{ matrix.extras }} --no-build-isolation ${{ matrix.pins }} "pygdal>=$(gdal-config --version).7,==$(gdal-config --version).*"
```

Comment:
```suggestion
          pip install -e .${{ matrix.extras }} --no-build-isolation ${{ matrix.pins }} "pygdal==$(gdal-config --version).*"
```

Reply:
that failed in https://github.com/nens/threedigrid-builder/actions/runs/4196155800